code/controllers/subsystems/jamming.dm

```diff
@@ -0,0 +1,24 @@
+// Used to check if a spot is currently being jammed
+// Can be used for telecomms or other stuff ,but right now its used for AI control
+
+SUBSYSTEM_DEF(jamming)
+	name = "Jamming"
+	init_order = INIT_ORDER_JAMMING
+	flags = SS_NO_FIRE
+	var/list/datum/component/jamming/active_jammers
+
+/datum/controller/subsystem/jamming/Initialize(start_timeofday)
+	active_jammers = new /list(world.maxz)
+	for(var/i = 1, i < world.maxz; i++)
+		active_jammers[i] = list()
+	. = ..()
+
+/datum/controller/subsystem/jamming/proc/IsPositionJammed(turf/location, signalStrength)
+	for(var/datum/component/jamming/jammer in active_jammers[location.z])
```

Comment:
`active_jammers` is declared as a list of jammer datums, but it looks like this is expecting it to be nested lists.